Polyfills/Console/Source/Console.cpp

```diff
@@ -34,16 +38,136 @@ namespace
         }
     }
 
+    bool EndsWith(std::string_view s, char c)
+    {
+        if (!s.empty())
+        {
+            return s.back() == c;
+        }
+        return false;
+    }
+
+    // based on: https://stackoverflow.com/a/26221725
+    template<typename T>
+    std::string FormatValue(std::string& formatString, T value)
+    {
+        // get the size of what will be written (+1 for null terminator)
+        int size_s = snprintf(nullptr, 0, formatString.c_str(), value) + 1;
```

Comment:
I know this is copied from stackoverflow, but I don't think you need +1 and then a subsequent -1 later. It used to be warranted for the buffer, but that's been replaced.

Reply:
I tried removing the +1/-1 when I noticed I didn't need the null terminator, but this gives me wrong results, of:
<img width="616" alt="result1" src="https://github.com/BabylonJS/JsRuntimeHost/assets/6002144/987bc11f-ebf5-492a-80f9-1d558b5cec27">
Opposed to:
<img width="392" alt="result2" src="https://github.com/BabylonJS/JsRuntimeHost/assets/6002144/ec1461fd-96fc-4d8a-a259-7be4be9bbdea">
With the +1/-1

Reply:
According to this doc https://cplusplus.com/reference/cstdio/snprintf/ it's not counting the null character.